packages/headless/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@coveo/headless",
   "private": false,
+  "homePage": "https://github.com/coveo/ui-kit/tree/master/packages/headless",
```

Comment:
In the case of headless, would it be better to point to the [docs page](https://docs.coveo.com/en/headless/latest/)? From the screenshot, it seems like there can be separate fields for "repository" vs. "homepage"?

Reply:
Indeed, we can do that.

There's two separate fields:
- https://docs.npmjs.com/cli/v7/configuring-npm/package-json#homepage
- https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository

I'll modify my PR accordingly.

Reply:
There is also this page for atomic: https://docs.coveo.com/en/atomic/latest/